identity/src/main/java/com/android/identity/util/Logger.kt

```diff
@@ -44,6 +29,9 @@ object Logger {
     private var fileWriterPath: String? = null
```

Comment:
I don't think modifying `Logger` is the right approach. This exists in the `identity` library which is also used on the server side and iOS where we don't have any Android dependcies.

We should have a separate `ActivityLogger` which lives in the `wallet` module. That way we can also easily use Android dependencies.